### PR TITLE
contribution guide: add EBNF preview and CustomContent instructions

### PR DESCRIPTION
**Pull request by qiancai**
### What is changed, added or deleted? (Required)

- Removed the Docs Dash information
- Added the guideline for contributing to TiDB Cloud documentation
- Added the guideline for previewing EBNF diagrams

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by dveeden — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,14 +4,7 @@ Welcome to [TiDB](https://github.com/pingcap/tidb) documentation! We are excited
...
-
-- **Dates/Time:** January 9 at 08:00 UTC ([your local time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=TiDB+Docs+Dash+2024%3A+Start&iso=20240109T08&p1=1440))  – January 12 at 07:59 UTC ([your local time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=TiDB+Docs+Dash+2024%3A+End&iso=20240112T0759&p1=1440))
-- **Event details**: <https://www.pingcap.com/event/tidb-docs-dash/>
-- **Issue list**: <https://github.com/orgs/pingcap/projects/51/views/3>
-- **Participation introduction**: <https://github.com/pingcap/docs/issues/15479>
-
-In addition to the issues and tasks in the event, you can also start from any one of the following items to help improve [TiDB Docs at the PingCAP website](https://docs.pingcap.com/tidb/stable):
+You can start from any one of the following items to help improve [TiDB Docs at the PingCAP website](https://docs.pingcap.com/tidb/stable):
```
Comment (qiancai) [inline CONTRIBUTING.md:7]:
```suggestion
You can start from any one of the following items to help improve [TiDB documentation at the PingCAP website](https://docs.pingcap.com/tidb/stable):
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -161,6 +154,59 @@ If your change fits one of the following situations, **CHOOSE THE AFFECTED RELEA
 - Fixes format to resolve a display error
 - Fixes broken links
 
+## Guideline for contributing to TiDB Cloud documentation
+
+Currently, the [TiDB Cloud documentation](https://docs.pingcap.com/tidbcloud/) is available only in English, and it is stored in the `release-7.5` branch of [pingcap/docs](https://github.com/pingcap/docs) for reusing SQL documents and development documents of TiDB v7.5. Hence, to create a pull request for TiDB Cloud documentation, make sure that your PR is based on the [release-7.5](https://github.com/pingcap/docs/tree/release-7.5) branch.
```
Comment (qiancai) [inline CONTRIBUTING.md:159]:
```suggestion
Currently, the [TiDB Cloud documentation](https://docs.pingcap.com/tidbcloud/) is available only in English, and it is stored in the [`release-7.5` branch](https://github.com/pingcap/docs/tree/release-7.5/tidb-cloud) of this repository for reusing SQL documents and development documents of TiDB v7.5. Hence, to create a pull request for TiDB Cloud documentation, make sure that your PR is based on the [release-7.5](https://github.com/pingcap/docs/tree/release-7.5) branch.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -161,6 +154,59 @@ If your change fits one of the following situations, **CHOOSE THE AFFECTED RELEA
 - Fixes format to resolve a display error
 - Fixes broken links
 
+## Guideline for contributing to TiDB Cloud documentation
+
+Currently, the [TiDB Cloud documentation](https://docs.pingcap.com/tidbcloud/) is available only in English, and it is stored in the [`release-7.5` branch](https://github.com/pingcap/docs/tree/release-7.5/tidb-cloud) of this repository for reusing SQL documents and development documents of TiDB v7.5. Hence, to create a pull request for TiDB Cloud documentation, make sure that your PR is based on the [release-7.5](https://github.com/pingcap/docs/tree/release-7.5) branch.
+
+> **Tip::**
```
Comment (qiancai) [inline CONTRIBUTING.md:161]:
```suggestion
> **Tip:**
```

**Review by lilin90 — APPROVED** [review]
/lgtm

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -161,6 +154,59 @@ If your change fits one of the following situations, **CHOOSE THE AFFECTED RELEA
 - Fixes format to resolve a display error
 - Fixes broken links
 
+## Guideline for contributing to TiDB Cloud documentation
+
+Currently, the [TiDB Cloud documentation](https://docs.pingcap.com/tidbcloud/) is available only in English, and it is stored in the [`release-7.5` branch](https://github.com/pingcap/docs/tree/release-7.5/tidb-cloud) of this repository for reusing SQL documents and development documents of TiDB v7.5. Hence, to create a pull request for TiDB Cloud documentation, make sure that your PR is based on the [release-7.5](https://github.com/pingcap/docs/tree/release-7.5) branch.
```
Comment (qiancai) [inline CONTRIBUTING.md:159]:
```suggestion
Currently, the [TiDB Cloud documentation](https://docs.pingcap.com/tidbcloud/) is available only in English, and it is stored in the [release-7.5](https://github.com/pingcap/docs/tree/release-7.5/tidb-cloud) branch of this repository for reusing SQL documents and development documents of TiDB v7.5. Hence, to create a pull request for TiDB Cloud documentation, make sure that your PR is based on the [release-7.5](https://github.com/pingcap/docs/tree/release-7.5) branch.
```

**Pull request merged**